.github/workflows/ci.yml

```diff
@@ -176,8 +183,21 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [ "3.7", "3.8", "3.9", "3.10", "3.11", "3.12" ]
+        python-version:
+          - "3.8"
```

Comment:
Ditto python 3.8 (end of support starts in 7 days)

Reply:
Idem.